### ISSUE TITLE
Review: layout/managers/a11y

### ISSUE THREAD

**Issue by fschutt** [fschutt/azul#396]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L31-L106

this entire section can be deleted

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L131

should be: feature = "a11y"

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L156-L158

Imports to the top

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L200-L217

Import NodeType at the top and simplify

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L235

Normalize text casing

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L283

normalize casing

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L324-L342

make a separate const fn for translations

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L501-L572

make a separate const translation function

**Comment by fschutt** [comment]
https://github.com/fschutt/azul/blob/d6ec01b9ddf91cfe0775a6b789b87f0e2fde8180/layout/src/managers/a11y.rs#L579

doc should be before cfg()
